src/test/java/jenkins/plugins/gerrit/workflow/GerritCommentStepTest.java

```diff
@@ -33,7 +33,7 @@ public void gerritCommentStepInvokeTest() throws Exception {
     p.setDefinition(
         new CpsFlowDefinition(
             "node {\n"
-                + "  withEnv(['BRANCH_NAME=21/5621/1']) {\n"
+                + "  withEnv(['GERRIT_API_URL=https://host/a/project', 'BRANCH_NAME=21/5621/1']) {\n"
```

Comment:
Shall we add a test then that when the GERRIT_API_URL isn't present, we have the error message in the job output?

Reply:
will try